### PR TITLE
Enable temporary disabled tests after release

### PR DESCRIPTION
**Pull request by vkozyura**

### CODE REVIEW

**Comment by vkozyura** [comment]
@danjoa: In case of a list we currently generate different cqn compared to the one obtained from compiler. See the test 'where ( ... cql  |  {x:y} )'. Please let me know if we should change it.

File: test/cds.ql.test.js [hunk 1]
```diff
@@ -261,7 +261,7 @@ describe('cds.ql → cqn', () => {
       // same for works distinct
     })
 
-    test.skip('where ( ... cql  |  {x:y} )', () => {
+    test('where ( ... cql  |  {x:y} )', () => {
       const args = [`foo`, "'bar'", 3]
```
Comment (danjoa) [inline test/cds.ql.test.js:265]:
```suggestion
      const args = [{ref:['foo']}, 'bar', 3]
```

File: test/cds.ql.test.js [hunk 2]
```diff
@@ -337,11 +335,31 @@ describe('cds.ql → cqn', () => {
...
+      const cqnFluent = {
+        SELECT: {
+          from: { ref: ['Foo'] },
+          where: [
+            { ref: ['ID'] },
+            '=',
+            { val: ID },
+            'and',
+            { ref: ['x'] },
+            'in',
```
Comment (danjoa) [inline test/cds.ql.test.js:352]: Fix to produce same output as compiler

File: test/cds.ql.test.js [hunk 3]
```diff
@@ -287,7 +286,7 @@ describe('cds.ql → cqn', () => {
                 'in',
                 { val: args },
```
Comment (danjoa) [inline test/cds.ql.test.js:287]: That seems completely wrong, should be something reflecting `args in (...)` while this one here says it's `args in 'args'`, which isn't even valid SQL
Fix to be same as by compiler.

File: test/cds.ql.test.js [hunk 4]
```diff
@@ -307,7 +305,7 @@ describe('cds.ql → cqn', () => {
...
```
Comment (danjoa) [inline test/cds.ql.test.js:306]: Same as above

**Comment by vkozyura** [comment]
> See inline comments

@danjoa: bli for the next sprint created

**Pull request closed without merging**